development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -35,23 +35,6 @@ func TestKymaIntegrationJobsPresubmit(t *testing.T) {
 				"installation/test/test/README.MD",
 			},
 		},
-		"Should contain the kyma-integration k3d with central Application Connectivity and Compass job": {
-			givenJobName: "pre-main-kyma-integration-k3d-central-app-connectivity-compass",
-
-			expPresets: []preset.Preset{
-				preset.GCProjectEnv, preset.KymaGuardBotGithubToken, preset.BuildPr, "preset-sa-vm-kyma-integration", "preset-kyma-integration-central-app-connectivity-enabled", "preset-kyma-integration-compass-dev", "preset-kyma-integration-compass-enabled",
-			},
-
-			expRunIfChangedRegex: "^((tests/fast-integration\\S+|resources\\S+|installation\\S+|tools/kyma-installer\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))",
-			expRunIfChangedPaths: []string{
-				"resources/values.yaml",
-				"installation/file.yaml",
-			},
-			expNotRunIfChangedPaths: []string{
-				"installation/README.md",
-				"installation/test/test/README.MD",
-			},
-		},
```

Comment:
I would rather avoid removing the current test cases as we don't have clear decision on how to move on with old job tests. Can you justify why did you remove this test case?

Reply:
I added back the unit test but reduced the scope of the test cases.